3.6/aql/functions-date.md

```diff
@@ -352,6 +352,36 @@ DATE_TRUNC('2017-02-03', 'month') // 2017-02-01T00:00:00.000Z
 DATE_TRUNC('2017-02-03 04:05:06', 'hours') // 2017-02-03 04:00:00.000Z
 ```
 
+### DATE_ROUND()
+
+`DATE_ROUND(date, amount, unit) → isoDate`
+
+Bin a date/time into a set of equal-distance buckets, to be used for
+grouping.
+
+- **date** (string\|number): a date string or timestamp
+- **amount** (number): number of *unit*s
+- **unit** (string): either of the following to specify the time unit (case-insensitive):
+  - y, year, years
```

Comment:
Looking at the code, I think `y`, `year` and `years` is not supported by `DATE_ROUND`.